dealership/vehicles.py

```diff
@@ -1,15 +1,35 @@
 class Vehicle(object):
+
     def __init__(self, maker, model, year, base_price, miles):
-        pass
+        self.maker = maker
+        self.model = model
+        self.year = year
+        self.base_price = base_price
+        self.miles = miles
 
 
 class Car(Vehicle):
-    pass
+
+    def sale_price(self):
```

Comment:
`sale_price` and `purchase_price` could be put in the parent class to provide a more generic method (if multipliers were class attributes as I mentioned in another comment) so that this method isn't repeated in every subclass